smithy-typescript-codegen/src/main/resources/software/amazon/smithy/typescript/codegen/protocol-test-stub.ts

```diff
@@ -137,17 +130,18 @@ const equivalentContents = (expected: any, generated: any): boolean => {
   }
 
   return true;
-}
+};
 
 const clientParams = {
   region: "us-west-2",
-  credentials: { accessKeyId: "key", secretAccessKey: "secret" }
-}
+  endpoint: "https://localhost/",
```

Comment:
This is the only change. The rest is formatting to match TypeScript code standards present in the repository.